smileidentity/web-client
# chore: reorganize components for publishing

This reorganises the components in web-components for publications.

## Testing
- Install dependencies `npm install`
- Run storybook `npm run storybook`
- View the stories to see nothing is broken
- Ran `npm publish --dry-run`
- Verify the files included
